students/shawn_degraw/session01/string-2.py

```diff
@@ -37,3 +37,28 @@ def end_other(a, b):
       return True
   return False
 
+#cat_dog
+def cat_dog(str):
+  catcount = 0
+  dogcount = 0
+  for i in range(len(str)-2):
+    comparestr = "cat"
+    nextstr = str[i:i+3]
+    if nextstr == comparestr:
```

Comment:
You might just have used `if str.startswith('cat', i):` without lines 45 & 46. You may refer to web pages like https://www.geeksforgeeks.org/python-string-startswith/ for that.